helm/kadalu/charts/operator/templates/resources.yaml

```diff
@@ -42,6 +43,23 @@ spec:
                 kadalu_format:
                   type: string
                   default: native
+                # Refer https://github.com/kubernetes-client/python/blob/da6076/kubernetes/docs/V1Toleration.md
+                tolerations:
+                  type: array
+                  items:
+                    type: object
+                    properties:
+                      effect:
+                        type: string
+                      key:
+                        type: string
+                      operator:
+                        type: string
+                      tolerationSeconds:
+                        format: int64
+                        type: integer
+                      value:
```

Comment:
Is the error/warning when "operator": "exists" and value is given, will this be handled internally by k8's?

Reply:
1. Schema is validated while applying CR as usual 
2. Toleration array is indirectly validated by k8s when patching the resource, it fails logging an error

Reply:
Ack.